tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -632,7 +629,14 @@ module.exports = {
       locateStrategy: 'xpath'
     },
     noResharePermissions: {
-      selector: '#oc-files-sharing-sidebar .files-collaborators-no-reshare-permissions-message'
+      selector:
+        '//div[@id="oc-files-sharing-sidebar"]//p[@data-test-id="files-collaborators-no-reshare-permissions-message"]',
+      locateStrategy: 'xpath'
+    },
+    noReshareLinkPermissions: {
+      selector:
+        '//div[@id="oc-files-file-link"]//p[@data-test-id="files-links-no-reshare-permissions-message"]',
```

Comment:
same as for the other one. The following selector should be sufficient:
```suggestion
        '//*[@data-test-id="files-links-no-reshare-permissions-message"]',
```